### PR TITLE
Ui login closes #30

### LINKED ISSUE #30: Improve the UI/UX

### PR DESCRIPTION
**Pull request by dawin09**

### CODE REVIEW

File: public/js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+(function() {
+    console.log("ready");
```
Comment (DannyFeliz) [inline public/js/scripts.js]: Remove this `console.log`

File: public/js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    if(WINDOWS_WIDTH > 768) {
+        AUTH_BG.setAttribute("src", `${UNSPLASH_API}${WINDOWS_WIDTH}x${WINDOW_HEIGHT}`);
+
+        return;
+    }
+
+    AUTH_BG.remove();
+
+    $(".intro-arrow").mouseover(function(){
+        console.log("hola");
```
Comment (DannyFeliz) [inline public/js/scripts.js]: Same here

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -11,73 +11,67 @@
...
+                    <a class="d-flex" href="{{ url('/') }}"><img src="/img/dashi-inline-logo.png" width="150" height="34" data-src-retina="assets/images/logo_white_2x.png" alt=""></a>
+                 </div>
+              </div>
+              <!-- <div class="pull-right">
+                 <div class="header-inner">
+                    <div class="visible-sm-inline visible-xs-inline menu-toggler pull-right p-l-10" data-pages="header-toggle" data-pages-element="#header">
+                       <div class="one"></div>
+                       <div class="two"></div>
+                       <div class="three"></div>
+                    </div>
```
Comment (DannyFeliz) [inline resources/views/layouts/app.blade.php]: Remove this zombie code

File: resources/views/welcome.blade.php [hunk 4]
```diff
@@ -9,97 +9,193 @@
...
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                    </div>
                 </div>
             </div>
         </div>
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```
Comment (DannyFeliz) [inline resources/views/welcome.blade.php]: Why we have multiples jquery?

File: public/js/jquery-1.11.1.min.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+/*! jQuery v1.11.1 | (c) 2005, 2014 jQuery Foundation, Inc. | jquery.org/license */
```
Comment (DannyFeliz) [inline public/js/jquery-1.11.1.min.js]: We already have jQuery 3, so remove this.

**Review by DannyFeliz — CHANGES_REQUESTED** [review]
Please fix this

File: public/js/scripts.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+(function() {
+    console.log("ready");
+    const AUTH_BG = document.querySelector(".auth-bg");
+    const WINDOWS_WIDTH = window.innerWidth;
+    const WINDOW_HEIGHT = window.innerHeight;
+    const UNSPLASH_API = "https://source.unsplash.com/category/technology/";
+
+    if(WINDOWS_WIDTH > 768) {
```
Comment (DannyFeliz) [inline public/js/scripts.js]: What is 768?

**Review by DannyFeliz — CHANGES_REQUESTED** [review]

**Comment by dawin09** [comment]
done, review the pr again

**Review by DannyFeliz — APPROVED** [review]

**Pull request merged**